sound/soc/sof/loader.c

```diff
@@ -821,8 +821,7 @@ int snd_sof_run_firmware(struct snd_sof_dev *sdev)
 	ret = snd_sof_dsp_run(sdev);
 	if (ret < 0) {
 		dev_err(sdev->dev, "error: failed to start DSP\n");
-		snd_sof_dsp_dbg_dump(sdev, SOF_DBG_DUMP_REGS | SOF_DBG_DUMP_MBOX |
-				     SOF_DBG_DUMP_PCI);
+		snd_sof_dsp_dbg_dump(sdev, SOF_DBG_DUMP_MBOX | SOF_DBG_DUMP_PCI);
```

Comment:
Why do we want to hide details in the event of a non FW boot. Wouldn't this be helpful in debugging ?

Reply:
These _were_ hidden as well before and at this point the DSP has not started to boot afaik. Not much to report about.

Reply:
Initially I only want to enable dumps from point where it certainly makes sense and we can expect valid information.